superdesk/vocabularies/vocabularies.py

```diff
@@ -444,6 +445,64 @@ def get_article_cv_item(self, item, scheme):
         article_item.update({"scheme": scheme})
         return article_item
 
+    def get_items(self, _id: str, qcode: str = None, is_active: bool = None,
+                  name: str = None, lang: str = None) -> Union[List, None]:
```

Comment:
might be easier for usage to return always a `List`, or `None` maybe if the cv doesn't exist, but not sure if that's needed